react-props-and-expressions/my-app/src/CustomButton.js

```diff
@@ -0,0 +1,7 @@
+export default function CustomButton({text, color}) {
+  return (
+    <>
```

Comment:
The `<>` aren't needed here since the JSX only has one root component. It's considered bad form to use `<>` when you don't need to. (But also not a big deal.)

Reply:
Thx Robert.  Will update.